Update Catalog Org TEP for Version Bump condition (#343)
tektoncd/community: teps/0003-tekton-catalog-organization.md

  - "@sthaha"
  - "@bobcatfish"
creation-date: 2020-06-11
last-updated: 2021-02-09

> status: implementable

Reviewer: Should we mark this as implemented btw ?

    3. Existing users of the resource can choose what versions of the
       resources they can use

> 3. Updates to already published resources should be released as a new versioned 

Reviewer 2: I think this may need some rephrasing,

```suggestion
If an update change the behavior of an already published tasks, the updated task should be a released as a new version.
```

>    task should be a released as a new version.

Reviewer 2: oops sorry my suggestion needs correction 
```suggestion
   task should be released as a new version.
```

  * When the image for a step is specified via an input params, we should treat changes to the default value of that param in the same way as changes to a step image

* New fields are added to the Task (new parameters, new results, new

>   workspace, new input, new output, …) and they are going to affect behaviour of the task which might break the pipeline created with existing version.

Reviewer 2: ```suggestion
  workspace, new input, new output, …) and they are going to affect the behavior of the task which might break the pipeline created with existing version.
```
